src/pkg/api/user.go

```diff
@@ -117,6 +120,20 @@ func (h *UserHandler) Register(c *gin.Context) {
 		return
 	}
 
+	if userCreateRequest.Role == uint8(roles.Patient) {
+		groupName := "doctors"
```

Comment:
mmm im really not shure about this, is this group very special? is it mean default groups for user with role patiient? do we have  info about this in docs?